ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -137,9 +138,13 @@ void RSkComponentTextInput::OnPaint(SkCanvas *canvas) {
       data.layoutManager->buildText(textLayout, textInputProps.backgroundColor, textInputProps.paragraphAttributes, textAttributes, displayString_, true);
     }
   }
-
   drawShadow(canvas, frame, borderMetrics, textInputProps.backgroundColor, layer()->shadowOpacity, layer()->shadowFilter);
   drawTextInput(canvas, component.layoutMetrics, textInputProps, textLayout);
+  if(hasToSetAutoFocus_){
+    requestForEditingMode(false);
+    hasToSetAutoFocus_ = false;
+    drawCursor(canvas, component.layoutMetrics);
```

Comment:
can this variable check be done in drawCursor function itself ? That way we will have drawCursor call at one place only...